pages/api/v1/answers.ts

```diff
@@ -468,17 +468,15 @@ async function Angela( // Good ol' Angela
   // When this number is reached, we remove the oldest messages from the context window
   const maxConvLength = model === "gpt-4-0613" ? 20 : 14;
 
-  // Store the system prompt in the feedback table so we can reconstruct it later
-  const { data, error } = await supabase.from("feedback").insert({
-    system_prompt: getMessages(
-      [],
-      actions,
-      reqData.user_description,
-      org,
-      language,
-    )[0].content,
-    conversation_id: conversationId,
-  });
+  if (redis) {
+    // Store the system prompt, in case we get feedback on it
+    await redis.set(
+      conversationId.toString(),
```

Comment:
it is overwritten yes. The system prompt is the same so it will be overwriting with the same value. GPT thinks this approach is correct.

<img width="804" alt="image" src="https://github.com/Superflows-AI/superflows/assets/28537547/cd29050c-78c0-41e3-9e8c-6bed485493eb">